vertx-opentracing/src/main/java/io/vertx/tracing/opentracing/VertxContextScopeManager.java

```diff
@@ -0,0 +1,114 @@
+package io.vertx.tracing.opentracing;
+
+import io.opentracing.Scope;
+import io.opentracing.ScopeManager;
+import io.opentracing.Span;
+import io.vertx.core.Context;
+import io.vertx.core.Vertx;
+
+/** VertxContextScopeManager is based on the {@link io.opentracing.util.ThreadLocalScopeManager}. */
+public class VertxContextScopeManager implements ScopeManager {
```

Comment:
Can we get around without this? I ideal would be to use only spans and where needed active them in the blocking execution to make them available in user-defined handlers.

Reply:
To be clear, you're advocating for removing the idea of a Context based scope manager? In implementing this, I was under the assumption that, given the potential for multiple contexts  on a given thread, it would be better to attach scopes to a context. Am I missing something?